DebugBle/BLE.cs

```diff
@@ -7,6 +7,12 @@
 
 public class BLE
 {
+    const string DLL_NAME =
+#if DEBUG
+        "BleWinrtDllDebug.dll";
+#else
+        "BleWinrtDll.dll";
+#endif
```

Comment:
![image](https://github.com/user-attachments/assets/28d210d1-5a51-4436-86ee-bfcf43191d90)

I could just be using it wrong, but even though I have my solution configuration set to "Debug", it is attempting to copy the non-debug version of the dll. 

Is this code snippet supposed to change which library is copied/loaded, or something else?

Reply:
After running it again this morning, it seems to have copied the correct file now 👍 , no clue what changed though!

![image](https://github.com/user-attachments/assets/c3be4263-55e8-40e1-b8f2-0d1ae89c43fe)

Just to confirm in Rider that the post-build event script is correct.